num_targets: 2
comm_dist: 2 
weights: [1.0, 1.0]
